tests/wallet/test_wallet.py

```diff
@@ -830,3 +830,90 @@ async def test_wallet_transaction_options(self, two_wallet_nodes, trusted):
 
         await time_out_assert(5, wallet.get_confirmed_balance, funds - AMOUNT_TO_SEND)
         await time_out_assert(5, wallet.get_unconfirmed_balance, funds - AMOUNT_TO_SEND)
+
+    @pytest.mark.parametrize(
+        "trusted",
+        [True, False],
+    )
+    @pytest.mark.asyncio
+    async def test_coin_selection_make_transaction(self, two_wallet_nodes, trusted):
```

Comment:
I'm not sure we need this test - but we need some tests of select_coins() itself to make sure the overall logic is correct. I see tests for the individual functions, but nothing for select_coins(). Some tests with very large lists might be worthwhile - some people will have 10s of thousands of dust coins, we should check that.

Reply:
Good point, after i separated the function i forgot to separate this.